src/m_class.c

```diff
@@ -919,22 +919,179 @@ t_pd *pd_newest(void)
     calls in typedmess(), to keep the compiler quiet. */
 typedef t_pd *(*t_newgimme)(t_symbol *s, int argc, t_atom *argv);
 typedef void(*t_messgimme)(t_pd *x, t_symbol *s, int argc, t_atom *argv);
+typedef void*(*t_messgimmer)(t_pd *x, t_symbol *s, int argc, t_atom *argv);
```

Comment:
Maybe a comment is good for these huge masses of typedefs. Maybe there is a way to have these in a separate file?